contracts/v0.8/types/CommonTypes.sol

```diff
@@ -64,5 +64,11 @@ library CommonTypes {
         bytes data;
     }
 
+    /// @param dataBytes deal proposal label in bytes format (it can be utf8 string or arbitray bytes string)
```

Comment:
Can we strengthen the documentation to indicate that this is actually an enum type and it's expect to be either or? Also check typos.

Reply:
I will add that in the inline docs